src/Components/ProvisioningWizard/steps/ReservationProgress/index.js

```diff
@@ -137,11 +139,17 @@ const ReservationProgress = ({ setLaunchSuccess }) => {
   return (
     <WizardContextConsumer>
       {({ goToStepById, onClose }) => (
-        <EmptyState variant="large">
-          <EmptyStateIcon color={(polledReservation?.success && '#3E8635') || null} icon={polledReservation?.success ? CheckCircleIcon : CogsIcon} />
-          <Title headingLevel="h4" size="lg" ouiaId="launch-status">
-            {polledReservation?.success ? 'System(s) launched successfully' : 'Launching system(s)'}
-          </Title>
+        <EmptyState variant="lg">
+          <EmptyStateHeader
+            headingLevel="h4"
+            title={polledReservation?.success ? 'System(s) launched successfully' : 'Launching system(s)'}
```

Comment:
I think it should be `titleText`. https://www.patternfly.org/components/empty-state#emptystateheader 
Because I am getting the following....
<img width="504" alt="Screenshot 2024-01-07 at 11 37 51 PM" src="https://github.com/RHEnVision/provisioning-frontend/assets/38880644/0839b963-4de5-4808-9bac-508e1056b73f">

Reply:
oh, good catch! :)